homework/index.js

```diff
@@ -30,14 +36,102 @@
     return elem;
   }
 
+  function createOptions(repoNames, select) {
+    createAndAppend('option', select, { text: 'Select module', disabled: 'disabled' });
```

Comment:
The assignment specified that you should display the details of the first repo in the (sorted) list at application startup. Therefore you should not add this placeholder option at the start of the list. (This in contrast to what we did in class).

At application startup the select element should display the name of the first repo (at present, `alumni`) and render the details of that repo in the page.